src/libraries/System.IO.Packaging/src/System/IO/Packaging/InternalRelationshipCollection.cs

```diff
@@ -47,16 +47,7 @@ IEnumerator IEnumerable.GetEnumerator()
         /// Returns an enumerator over all the relationships for a Package or a PackagePart
         /// </summary>
         /// <returns></returns>
-        IEnumerator<PackageRelationship> IEnumerable<PackageRelationship>.GetEnumerator()
-        {
-            return _relationships.GetEnumerator();
-        }
-
-        /// <summary>
-        /// Returns an enumerator over all the relationships for a Package or a PackagePart
-        /// </summary>
-        /// <returns></returns>
-        public List<PackageRelationship>.Enumerator GetEnumerator()
+        public IEnumerator<PackageRelationship> GetEnumerator()
```

Comment:
Previously foreach'ing over an InternalRelationshipCollection would be non-allocating; now it'll result in an enumerator being allocated.  Are there any places where that was being done?

Reply:
The only place this was used was promptly cast to an `IEnumerator` to be filtered elsewhere.